src/app/space/create/deployments/apps/deployment-status-icon.component.spec.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  ComponentFixture,
+  TestBed
+} from '@angular/core/testing';
+
+import { By } from '@angular/platform-browser';
+import {
+  Component,
+  DebugElement,
+  Input
+} from '@angular/core';
+
+import { Observable } from 'rxjs';
+import { DeploymentStatusIconComponent } from './deployment-status-icon.component';
+
+import { CpuStat } from '../models/cpu-stat';
+
+const ICON_OK = 'pficon-ok';
+const ICON_WARN = 'pficon-warning-triangle-o';
+const ICON_ERR = 'pficon-error-circle-o';
+const MSG_OK = 'Everything is ok.';
+
+describe('DeploymentStatusIconComponent', () => {
+
+  let component: DeploymentStatusIconComponent;
+  let fixture: ComponentFixture<DeploymentStatusIconComponent>;
+  let stat: CpuStat;
+
+  let mockCpuData = Observable.of({ used: 1, total: 5 } as CpuStat) ;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
```

Comment:
Can you use initContext/TestContext here? Fine to do in a followup patch. There are examples around the Deployments area but please feel free to ask for help if you're unsure about how to use it or what it provides.